classes/v1migration/v1migration.php

```diff
@@ -102,46 +102,17 @@ public static function output_progress_bar() {
     }
 
     /**
-     * Return the true if the user proceeds with the migration.
+     * Since April 2019 the parameters taken in are not used, however we leave them in to avoid forcing a V1 release/update.
      *
      * @param int $courseid - The course ID.
      * @param int $turnitintooltwoid - The turnitintooltwoid.
-     * @return string $output The HTML for the modal.
      */
     public function migrate_modal($courseid, $turnitintoolid) {
         global $PAGE;
-        $cssurl = new moodle_url('/mod/turnitintooltwo/jquery/colorbox.css');
-        $PAGE->requires->css($cssurl);
         $cssurl = new moodle_url('/mod/turnitintooltwo/css/font-awesome.min.css');
         $PAGE->requires->css($cssurl);
-        $PAGE->requires->jquery_plugin('turnitintooltwo-colorbox', 'mod_turnitintooltwo');
-        $PAGE->requires->jquery_plugin('turnitintooltwo-turnitintooltwo', 'mod_turnitintooltwo');
-        $PAGE->requires->jquery_plugin('turnitintooltwo-migration_tool', 'mod_turnitintooltwo');
-
-        $PAGE->requires->string_for_js('closebutton', 'turnitintooltwo');
-
-        $migratelink = html_writer::tag('button', get_string('migrateassignment', 'turnitintooltwo'),
-                                                    array('class' => 'migrate_link btn-primary', 'id' => 'migrate_link',
-                                                    'data-courseid' => $courseid, 'data-turnitintoolid' => $turnitintoolid));
-        $dontmigratelink = html_writer::tag('button', get_string('dontmigrateassignment', 'turnitintooltwo'),
-                                                        array('class' => 'dontmigrate_link btn-default', 'id' => 'dontmigrate_link'));
-
-        $spinner = html_writer::tag('i', '', array('class' => 'fa fa-spinner fa-spin fa-2x'));
-        $spinner = html_writer::tag('div', $spinner, array('class' => 'migration-spinner'));
 
-        $migrating = html_writer::tag('div', html_writer::tag('p', get_string('migrating', 'turnitintooltwo'))
-                                        . $spinner
-                                        . html_writer::tag('p', get_string('migrationredirect', 'turnitintooltwo')),
-                                        array('id' => 'migrating', 'class' => 'hide'));
-
-        $asktomigrate = html_writer::tag('div', html_writer::tag('p', get_string('migrationtooltitle', 'turnitintooltwo'), array('class' => 'migrationtitle'))
-                                        . html_writer::tag('p', get_string('migrationtoolinfo', 'turnitintooltwo'), array('class' => 'migrationtoolinfo'))
-                                        . $migratelink . $dontmigratelink
-                                        , array('id' => 'asktomigrate', 'class' => 'hide'));
-
-        $output = html_writer::tag('div', $asktomigrate . $migrating, array('id' => 'migration_alert', 'class' => 'hide'));
-
-        return $output;
```

Comment:
Does this still expect a return in v1? If so, is it going to cause an issue returning an empty string?

Reply:
In V1 it just `echo`s the result out. Having no return on an echo doesn't appear to cause any issues or undefined messages, but I could return `''` if it's safer.